shu-etal.ipynb

```diff
@@ -0,0 +1,1516 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "525fe7fa",
+   "metadata": {},
+   "source": [
+    "# Replication: Shu *et al*, 2020\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook attempts to replicate the following paper with the [PPMI](http://ppmi-info.org) dataset:\n",
+    "\n",
+    "<div class=\"alert alert-block alert-success\">\n",
+    "Shu, Zhen‐Yu, et al. <a href=\"https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.28522?casa_token=Ab53WvMlODcAAAAA%3AXcgDLmq8egqW7uwd2g3jY9jIljhLu3VhIbvMWgbcfoWOxjO_9H7Arf91t2FBZDZ8E94Je4Wmrn0ZmkeZ\">Predicting the progression of Parkinson's disease using conventional MRI and machine learning: An application of radiomic biomarkers in whole‐brain white matter.</a> Magnetic Resonance in Medicine 85.3 (2021): 1611-1624.</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "078e819a",
+   "metadata": {},
+   "source": [
+    "This study recruited 72 patients with progressive PD and 72 patients with stable PD matched by age, sex and category of HYS. This following population from PPMI was used:\n",
+    "\n",
+    "<img src=\"./images/table.png\" width=\"80%\"/>\n",
+    "\n",
+    "The remainder of this notebook is an attempt to replicate this result using the PPMI dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b81bcec8",
+   "metadata": {},
+   "source": [
+    "## Initial setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed06fa75",
+   "metadata": {},
+   "source": [
+    "LivingPark notebooks use a *cache* directory to store analysis inputs and outputs. Inputs typically include PPMI Study Data and imaging data whereas outputs include processed images and other derivatives. Cache directories allow LivingPark notebooks to run in a few minutes as they reuse previously computed results. However, cache directories cannot be made public due to the PPMI Data Usage Agreement (DUA). Instead, they are stored on `login.bic.mni.mcgill.ca`, which requires a specific user name and password as well as a properly-configured DataLad installation in your account (see instructions [here](https://docs.google.com/document/d/1K7RzQjYC1O6tpSqe_c0CvxvJq38J10zNrDIdw20Yh_0/edit?usp=sharing)). In case you don't have access to the cache directory of this notebook, the next sections will download all the required imaging data from PPMI and recompute the results, which will take a few hours depending on your computer configuration. In the future, we will aim at storing this cache dataset on PPMI servers so that they can be accessed with a PPMI account.\n",
```

Comment:
The cache directory currently isn't functional, let's update this comment accordingly.